server/scripts/download_batch_output.py

```diff
@@ -0,0 +1,35 @@
+
+from dotenv import load_dotenv
+from openai import OpenAI
+import time
+import os
+import json
+
+
+load_dotenv()
+OPENAI_API_KEY = os.getenv("OPENAI_API_KEY")
+
+openAI_client=OpenAI(api_key=OPENAI_API_KEY)
+
+time.sleep(5)
+downloaded=[]
+
+with open('batch_jobs_created.txt', 'r') as file:
+    batch_jobs = json.loads(file.read())
+    file.close()
+
+count=0
+while len(downloaded)<len(batch_jobs):
+    print(f'Try: {count}')
+    for i in batch_jobs:
+        if  i not in downloaded:
+            job=openAI_client.batches.retrieve(i)
```

Comment:
Repetetive code, can update and use : def download_file code in services/openai_client.py